### ISSUE TITLE
Alias/Abbeviation naming mapping feature request

### ISSUE THREAD

**Issue by williamwsmith** [kkozmic/Cartographer#14]
One of the difficulties I have found with AutoMapper is not enough flexibility in name matching.  Quite often in large older systems, the are abbreviations/aliases used.  Typicall the new code usings more literate names.  Would be nice to have a set of aliases/abbreviates that could be applied to generate match candidates.

example 
from name:  PKG_TRKNO
to name: PackageTrackingNumber

Automapper does support prefixes. (PKG_) in above.
Would like to register- "PKG_", as "Package" and "TRKNO" as "TrackingNumber".  Just a thought.  Glad to see you are taking this on!

**Comment by kkozmic** [comment]
Cheers,

I can see how that can be useful, although I think I'd rather provide crazy simple way to define those synonyms/aliases/abbreviations than try to have any out of the box ones, since I think that would be pretty domain/application specific

**Comment by williamwsmith** [comment]
Agreed.  I didn't mean to imply having having common abbreviations in your implementation was the solution.
 
W.

**Comment by kkozmic** [comment]
Ah ok.

That's what I thought but wanted to make it very clear :)

@K
